bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/RuleStatusInfo.java

```diff
@@ -69,6 +74,7 @@ public RuleStatusInfo(RuleStatus status, RuleStatusDetail statusDetail, String d
         this.status = status;
         this.statusDetail = statusDetail;
         this.description = description;
+        this.properties = properties;
```

Comment:
The RuleStatusInfo of triggered state should contain output values of the trigger which was activated the rule. These values can be used by a CEP trigger based on the existing trigger type, also  they can be used for logging or remote tracking of automation engine.